docs/upgrade/upgrade-guides/upgrade-guide-from-2024.x-to-2025.1/upgrade-guide-from-2024.x-to-2025.1.rst

```diff
@@ -305,13 +270,14 @@ Download and install the old release
 
    .. group-tab:: Debian/Ubuntu
```

Comment:
.. group-tab:: EC2/GCP/Azure Ubuntu Image 

is missing here, with installing old scylla-machine-image

Reply:
it is tested, and we always supported it.
it's a must, in case of any failure that a user can rollback a node.
we don't support rollback after all nodes were upgrade.

images upgrade/rollback are identical to regular upgrade/rollback, but there only extra package that need to be mentioned